dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tool.java

```diff
@@ -143,6 +146,11 @@
     @Cascade(CascadeType.DETACH)
     private final SortedSet<Tag> workflowVersions;
 
+    @JsonIgnore
+    @OneToOne(targetEntity = Tag.class, fetch = FetchType.LAZY)
+    @JoinColumn(name = "actualDefaultVersion", referencedColumnName = "id")
```

Comment:
Why call this actual default version instead of just default version?

Reply:
it already exists